docs/userGuide/cliCommands.md

```diff
@@ -93,6 +93,13 @@ Essentially, this optional feature is very useful when writing content, more so
 The caveat is that not building all pages during the initial process, or not rebuilding all affected pages when a file changes, will cause your search results for these pages to be empty or outdated, until you navigate to them to trigger a rebuild.
 </box>
 
+* `-b`, `--background-build` **[BETA]**<br>
+   If `--one-page` is specified, this mode enhances the single-page serve by building the pages that are not yet built
+   or marked to be rebuilt in the background.
+   
+   You can still edit the pages during the background build. When MarkBind detects changes to the source
+   files, the background build will stop, rebuild the files affected, then resumes the background build with the
+   remaining pages.
```

Comment:
@ryoarmanda We can mention what's the benefit to the user. Otherwise there is no motivation for the user to use this option.